src/BizHawk.Emulation.Cores/Arcades/MAME/MAME.cs

```diff
@@ -174,21 +92,21 @@ private void ExecuteMAMEThread()
 				, "-skip_gameinfo"                      // forbid this blocking screen that requires user input
 				, "-nothrottle"                         // forbid throttling to "real" speed of the device
 				, "-update_in_pause"                    // ^ including frame-advancing
-				, "-rompath",            _gameDirectory // mame doesn't load roms from full paths, only from dirs to scan
+				, "-rompath",                      ""   // mame doesn't load roms from full paths, only from dirs to scan

```

Comment:
Legend?

Reply:
The absolute path within wbx's virtual filesystem for the ROM is whatever I set in the AddReadonlyFile call. Which would be the rom filename itself. So rompath needs to be "".

Reply:
Does it launch without `-rompath`?

Reply:
nope